state-chain/node/src/chain_spec/common.rs

```diff
@@ -3,7 +3,7 @@ use sp_runtime::{Percent, Permill};
 pub use state_chain_runtime::constants::common::*;
 use state_chain_runtime::{chainflip::Offence, BlockNumber, FlipBalance, SetSizeParameters};
 
-pub const GENESIS_FUNDING_AMOUNT: FlipBalance = 5_000 * FLIPPERINOS_PER_FLIP;
```

Comment:
why are we changing this amount?

Reply:
@albert-llimos mentioned it on discord - it's 1000 FLIP in the Sepolia contracts.

Reply:
Albert already funded the accounts with 1K